File: Example/WZRootNavigationController_Example_ObjC/ViewControllers/WZStatusBarHiddenViewController.m
//
//  WZStatusBarHiddenViewController.m
//  WZRootNavigationController
//
//  Created by 吴哲 on 2018/4/25.
//  Copyright © 2018年 CocoaPods. All rights reserved.
//

#import "WZStatusBarHiddenViewController.h"

@interface WZStatusBarHiddenViewController ()

@end

@implementation WZStatusBarHiddenViewController

- (BOOL)prefersStatusBarHidden{
    return  YES;
}

-(UIStatusBarAnimation)preferredStatusBarUpdateAnimation
{
    return  UIStatusBarAnimationFade;
}
@end

File: Example/WZRootNavigationController/ViewControllers/WZStatusBarHiddenViewController.swift
//
//  WZStatusBarHiddenViewController.swift
//  WZRootNavigationController
//
//  Created by 吴哲 on 2018/4/23.
//  Copyright © 2018年 CocoaPods. All rights reserved.
//

import UIKit

class WZStatusBarHiddenViewController: UIViewController {

    override var prefersStatusBarHidden: Bool{
        return true
    }
    
    override var preferredStatusBarUpdateAnimation: UIStatusBarAnimation{
        return .slide
    }

}
